config/config.md

```diff
@@ -102,16 +102,19 @@ The following config option are provided by the OpenHIM. All of these options ha
     "enableBasicAuthentication": true,
     "enableCustomTokenAuthentication": false,
     "enableJWTAuthentication": false,
```

Comment:
I accidentally committed my testing changes. I meant to make the default `false` as this may cause setup issues for unsuspecting users.
Users might upgrade their existing OpenHIM instances and not realize that JWTs are suddenly enabled on their instance, but without proper configuration.
Also, even if we do enable this by default along with default config this would be a security issue as there's no way to easily update the JWT default config - as we do with the default Basic Auth credentials on the first login.

Reply:
ok cool that makes sense. Lets keep it disabled by default and allow the implementer to configure it properly if they want to use it